### PR TITLE
change product names

### PR DESCRIPTION
**Pull request by technomerlyn**

### CODE REVIEW

File: logging/Limiting_Access_to_Logs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ only for a specified namespace.
 each tenant administrator's access so that they can access the 
 log information only for their SAS Viya tenant.
 
-This access is controlled by a combination of Kibana-tenant spaces and 
+This access is controlled by a combination of OpenSearch Dashboards-tenant spaces and 
 Open Distro for Elasticsearch roles, back-end roles, and role mappings.
```
Comment (gsmith-sas) [inline logging/Limiting_Access_to_Logs.md:30]: Replace "ODFE" with "OpenSearch" on line 30.

File: logging/Limiting_Access_to_Logs.md [hunk 2]
```diff
@@ -137,25 +136,25 @@ logging/bin/offboard.sh --namespace <i>namespace</i>  [--tenant <i>tenant</i>]
...
+combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the OpenSearch Dashboards-tenant space named `mynamespace`. If you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1`, the script would remove the OpenSearch Dashboards-tenant space named `mynamespace_mytenant1`. 
 
 ## Managing User Accounts for Restricted Access
 
 ### Introduction
 
-You can create or delete Kibana accounts 
-that have access to only the log messages and Kibana-tenant space 
+You can create or delete OpenSearch Dashboards accounts
```
Comment (gsmith-sas) [inline logging/Limiting_Access_to_Logs.md:151]: My bad.  The accounts are technically "OpenSearch accounts" and NOT "OpenSearch Dashboards accounts".  The tenant spaces are "OpenSearch Dashboard-tenant spaces"

File: logging/Tenant_Logging.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ for a SAS Viya tenant.
 Use the `/logging/bin/offboard.sh` script to remove logging access 
 for a SAS Viya tenant.
 
-Use the `/logging/bin/user.sh` script to create or delete Kibana accounts 
-that have access to only the log messages and Kibana tenant space 
+Use the `/logging/bin/user.sh` script to create or delete OpenSearch Dashboards accounts
```
Comment (gsmith-sas) [inline logging/Tenant_Logging.md:17]: "OpenSearch accounts" rather than "OpenSearch Dashboards accounts"

File: logging/README.md [hunk 4]
```diff
@@ -105,7 +115,7 @@ After you create the location for your customization files, you can customize th
 my-viya4mon-user-dir/user.env
 
 my-viya4mon-user-dir/logging/user.env
-my-viya4mon-user-dir/logging/user-values-elasticsearch-open.yaml
+my-viya4mon-user-dir/logging/user-values-opensearch-open.yaml
```
Comment (gsmith-sas) [inline logging/README.md:118]: File name should be: my-viya4mon-user-dir/logging/user-values-opensearch.yaml

File: logging/README.md [hunk 5]
```diff
@@ -179,19 +193,17 @@ make in the `user-values-fluent-bit-open.yaml` file that are intended to
...
+When you edit the `user-values-opensearch.yaml` file, ensure that the parent
 item of any item that you uncomment is also uncommented.  For
-example, if you uncommented the `storageClass` item for the Elasticsearch
+example, if you uncommented the `storageClass` item for the OpenSearch
 master nodes, you must also uncomment the `persistence` item, the
 `master` item and the `elasticsearch` item, as shown below:
 
 ```yaml
-# Sample user-values-elasticsearch-open.yaml
+# Sample user-values-opensearch-open.yaml
```
Comment (gsmith-sas) [inline logging/README.md:203]: Change to: # Sample user-values-opensearch.yaml

**Review by gsmith-sas — CHANGES_REQUESTED** [review]

**Review by cumcke — APPROVED** [review]
Looks good to me.

File: logging/Limiting_Access_to_Logs.md (hunk 1 above)
Comment (technomerlyn) [reply logging/Limiting_Access_to_Logs.md:30]: Done

File: logging/Limiting_Access_to_Logs.md (hunk 2 above)
Comment (technomerlyn) [reply logging/Limiting_Access_to_Logs.md:151]: Corrections made.

File: logging/README.md (hunk 4 above)
Comment (technomerlyn) [reply logging/README.md:118]: Correction made.

File: logging/README.md (hunk 5 above)
Comment (technomerlyn) [reply logging/README.md:203]: Correction made.

**Pull request merged**